net/frr/src/opnsense/mvc/app/models/OPNsense/Quagga/OSPF6.xml

```diff
@@ -13,6 +13,7 @@
                 <default></default>
                 <OptionValues>
                         <connected>Connected routes (directly attached subnet or host)</connected>
+                        <kernel>Kernel routes (not installed via the zebra RIB)</kernel>
```

Comment:
Have you really verified that this works?

Reply:
It's actually a required setting for OSPF6 to redistribute default routes if you use gateways with OPNsense: that's the change that got me to even create the issue in the first place.